parachains/runtimes/assets/westmint/src/lib.rs

```diff
@@ -669,9 +678,20 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckingAccount>,
+	// Note: Removed v1 storage migration. Assume this has been applied in some upgrade.
+	// Do not release until verified!
+	MigrateAssetsPallet,
 >;
 
+pub struct MigrateAssetsPallet;
+impl frame_support::traits::OnRuntimeUpgrade for MigrateAssetsPallet {
+	fn on_runtime_upgrade() -> Weight {
+		use frame_support::storage::migration;
+		migration::move_pallet(b"Assets", b"TrustBackedAssets");
+		<Runtime as frame_system::Config>::DbWeight::get().writes(1)
```

Comment:
The weight is not really correct :P

Reply:
Yeah, I was thinking of removing the rename because it will break most wallets/interfaces that read from Statemint.

Reply:
Yeah also sounds reasonable. I mean we can just name the new ones appropriate.